_posts/2021-03-30-influxdb-rails.html

```diff
@@ -0,0 +1,199 @@
+---
+image: fleur-dQf7RZhMOJU-unsplash.jpg 
+image_license: Photo by Fleur on <a href="https://unsplash.com/photos/dQf7RZhMOJU">Unsplash</a>
+title: Measure twice, instrument once with influxdb-rails!
+description: A blog post about the Ruby on Rails APM with influxdb-rails
+---
+
+<p>
+  Now that we learned how truly magnificent <code>ActiveSupport::Notifications</code> is in the previous post,
+  let's explore a RubyGem <a href="https://bruckmayer.net/">Chris</a>, others and me have built around this: influxdb-rails.
+  Together with some other awesome Free Software, it will help you to deep dive into and understand your Ruby on Rails
+  application performance.
+</p>
+
+<div id="more"></div>
+
+<div class="alert alert-warning">
+  <p>
+    <small>
+      This post ist part two of a three part series about instrumenting your Ruby on Rails app with Active Support Instrumentation
+      and influxdb-rails. Make sure to check them all out!
+    </small>
+    <ul>
+      <li>
+        <a href="/blog/activesupport.html">ActiveSupport::Notifications is rad! 💘</a>
+      </li>
+      <li>
+        Measure twice, instrument once with influxdb-rails! (this post)
+      </li>
+      <li>
+        <a href="/blog/beyond-performance.html">influxdb-rails instrumentation beyond performance.</a>
+      </li>
+    </ul>
+  </p>
+</div>
+
+<h2>Application Performance Monitoring (APM)</h2>
+<p>
+  I'm not sure we have to discuss this at all anymore, but here is why I think I need application performance
+  monitoring for my Ruby on Rail apps: I'm an expert in using the software I hack. Hence I always travel the happy
+  path, cleverly and unconsciously avoid the pitfalls, use it with forethought of the underlying architecture.
+  Because of that, I usually <b>think</b> my app is fast: <a href="https://blog.teamtreehouse.com/perceived-performance">perceived performance</a>.
+</p>
+
+<p>
+  <blockquote>
+    It's not what you look at that matters, it's what you see.
+    – Henry David Thoreau
+  </blockquote>
+</p>
+
+<p>
+  That is why I need someone to correct that BIAS for me, in black and white.
```

Comment:
I don't really understand why _bias_ is capitalized.

```suggestion
  That is why I need someone to correct that bias for me, in black and white.
```